sdcm/cluster.py

```diff
@@ -1268,12 +1268,13 @@ def is_manager_server_up(self, port=None):
         http_status_code = int(curl_output.stdout.strip())
         return http_status_code == 204
 
-    def wait_manager_server_up(self, verbose=True, timeout=300):
+    def wait_manager_server_up(self, verbose=True, timeout=300, port=None):
```

Comment:
I don't see any used with the `port` variable...

Reply:
I used the variable two lines below....

Reply:
Sorry, I missed it...